src/components/GlobalPreferences/CustomLabels/LocalIdentities.js

```diff
@@ -0,0 +1,367 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {
+  Box,
+  Button,
+  ButtonIcon,
+  Checkbox,
+  GU,
+  IconArrowDown,
+  IconArrowUp,
+  IconDown,
+  IconDownload,
+  IconExternal,
+  IconGrid,
+  IconSearch,
+  IconShare,
+  IconTrash,
+  IdentityBadge,
+  Info,
+  TextInput,
+  breakpoint,
+  font,
+  useTheme,
+} from '@aragon/ui'
+import EmptyFilteredIdentities from './EmptyFilteredIdentities'
+import ButtonDropDown from '../../ButtonDropDown/ButtonDropDown'
+import LocalIdentityPopoverTitle from '../../IdentityBadge/LocalIdentityPopoverTitle'
+import { fileImport } from './Import'
+import { ASC, DESC } from './useLocalIdentities'
+
+const iOS = /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream
+
+function LocalIdentities({
+  allSelected,
+  identities,
+  identitiesSelected,
+  onClear,
+  onExport,
+  onImport,
+  onRemove,
+  onSearchChange,
+  onShare,
+  onShowLocalIdentityModal,
+  onToggleAll,
+  onToggleIdentity,
+  onToggleSort,
+  searchTerm,
+  someSelected,
+  sortIdentities,
+}) {
+  const theme = useTheme()
+  const inputRef = React.useRef()
+  // trigger file chooser
+  const handleImportClick = () => inputRef.current.click()
```

Comment:
Ahh, yes, I took it away because the button was part of the `ButtonDropDown` options, I'll bring it back - Chrome natively already supports dropping files in inputs.

Reply:
Yes most browsers do AFAIK, but what I meant is that maybe we could add a drop zone (that would cover the entire area)?

Reply:
The functionality was re-implemented for the button, I think this comment is resolved, if we want to add a feature like the one you describe maybe we can follow up someplace else?
cc @dizzypaty about the new feature?

Reply:
Thanks @AquiGorka to add this functionality to the button again. Not sure people will try to drop files on the button, is too small as a drop area and pretty undiscoverable if we don't apply a visual treatment to it. The entire table (or even the screen) should become the drop area. However I think we can do this later, after 0.8. I'll make a note of this and design the screen :)